samples/hello-android/settings.gradle

```diff
@@ -1 +1,15 @@
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        maven { url "https://jcenter.bintray.com/" }
+    }
+    resolutionStrategy {
+        eachPlugin {
+            if (requested.id.id == "com.android.application") {
+                useModule("com.android.tools.build:gradle:${requested.version}")
+            }
+        }
+    }
+}
+
 rootProject.buildFileName = 'build.gradle.kts'
```

Comment:
Is this line required anymore? Isn't this autodetected now?

Reply:
@JLLeitschuh it is still required due to the existence of `build.gradle` (required by Android Studio).

Reply:
Ah. Fair enough.